src/librustdoc/html/static/themes/dark.css

```diff
@@ -364,16 +364,20 @@ kbd {
 	background: #f0f0f0;
 }
 
+#theme-picker:hover {
```

Comment:
Can you add the same style for `:focus`? Currently, you can't tell if the button is focused or not.

Reply:
Sure.